hs_discover/src/components/Resources.vue

```diff
@@ -337,12 +335,21 @@ export default {
       this.searchtext = document.getElementById('qstring').value.trim();
     }
     this.searchClick(false, true);
-    // this.filterBuilder();
   },
   methods: {
-    searchClick(paging, dofilters) { // paging flag to skip the page reset after data retrieval
+    searchClick(paging, dofilters, reset) { // paging flag to skip the page reset after data retrieval
       if (!this.pagenum) return; // user has cleared input box with intent do manually input an integer and subsequently caused a search event
       document.body.style.cursor = 'wait';
+      if (reset) {
+        this.startdate = '';
+        this.enddate = '';
+        this.authorFilter = [];
+        this.ownerFilter = [];
+        this.subjectFilter = [];
+        this.contributorFilter = [];
+        this.typeFilter = [];
+        this.availabilityFilter = [];
+      }
       axios.get('/discoverapi/', {
```

Comment:
To satisfy all possible sequences / workflows clicking around, a freetext search should reset all previous selections, then perform a new search and redo integer counts beside checkboxes